winui/Rating/Clear-rating.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: Clear rating value in WinUI Rating control | Syncfusion
+description: Learn all about how to clear the rating value in the WinUI Rating (SfRating) control.
+platform: WinUI
+control: SfRating
```

Comment:
Control: SfRating

Reply:
We have follow the small letter for all other controls.